### PR TITLE
Add timestamp to the message. Close #16

### LINKED ISSUE #16: show timestamp in conversation messages

### PR DESCRIPTION
**Pull request by soueuls**
This is my attempt at adding support for timestamp message.
I also added a CSS class so the user can further customize.

This is my first public PR, let me know if there is anything I can do.

### CODE REVIEW

File: src/types.ts [hunk 1]
```diff
@@ -49,6 +49,7 @@ export interface ChatOptions {
   doneMultipleChoiceLabel?: string;
   andSeparatorText?: string;
   submitLabel?: string;
+  showTimestamp?: boolean;
```
Comment (andersonba) [inline src/types.ts]: `timestampable`

**Comment by andersonba** [comment]
Fix the tests, also, create one to ensure this new feature

**Comment by soueuls** [comment]
Thank you for the feedback ! I will happily work on this tonight or tomorrow, i definitely like the date formatter idea better.

**Comment by andersonba** [comment]
Awesome, @soueuls 🤘
I need you to rebase your branch with master to merge this PR. Remember to re-build the lib.

**Comment by soueuls** [comment]
I rebased from master, build the files.
But then I messed things up when I tried to squash the commits, I hope you can fix the historic to make it looks a bit nicer.

I am still learning few things with git 🤕

**Comment by andersonba** [comment]
hahaha that's ok, no problem. I will squash and merge your PR.
Check the [tests](https://travis-ci.org/andersonba/yve-bot/builds/290948324), there is a broken test.
You need to mock the date using `jest.useFakeTimers`, [see here](https://facebook.github.io/jest/docs/en/timer-mocks.html#content).

**Pull request merged**